src/Form/Form.js

```diff
@@ -0,0 +1,64 @@
+import "./Form.css";
+import React, { useState } from "react";
+
+export default function Form() {
+  const [meme, setMeme] = React.useState({
```

Comment:
The use of `React.useState` is overkill.
- `useState` has already been imported from react > Review object destructuring
-  use `const [meme,setMeme] = useState({...})`
- You can remove the React import. JSX is automatically interpreted with the late react release
> `import {useState,useEffect} from "react"`

- [ ]  Replace all instances of `React.method()` with just the `method()`

Reply:
Removed and Changed according to request